### PR TITLE
Userstory1 api get

### PR DESCRIPTION
**Pull request by Wojciech-Skarbek-K**

### CODE REVIEW

**Review by MonikaKucinska — APPROVED** [review]

File: src/main/java/org/kainos/groupA/api/JobRoleService.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.sql.SQLException;
+import java.util.List;
+
+@Path("/api")
+@Api("User API")
+public class JobRoleService {
```
Comment (shaunganley) [inline src/main/java/org/kainos/groupA/api/JobRoleService.java:17]: this is a controller class, not a service

File: src/main/java/org/kainos/groupA/services/JobRoleService.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.sql.SQLException;
+import java.util.List;
+
+@Path("/api")
+@Api("User API")
```
Comment (shaunganley) [inline src/main/java/org/kainos/groupA/services/JobRoleService.java:16]: these annotations not needed

File: src/test/java/org/kainos/groupA/integration/JobRoleIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            GroupAApplication.class, null,
+            new ResourceConfigurationSourceProvider()
+    );
+
+    @Test
+    void getJobRoles_shouldReturnListOfJobRoles() {
+       List<JobRole> response = APP.client().target("http://localhost:8080/api/job-roles")
+               .request()
+               .get(List.class);
+        Assertions.assertTrue(response.size() > 0);
```
Comment (shaunganley) [inline src/test/java/org/kainos/groupA/integration/JobRoleIntegrationTest.java:27]: can you assert that the first element in the list has a role name that is not null?

**Review by shaunganley — COMMENTED** [review]
2 small comments

**Review by shaunganley — APPROVED** [review]

**Pull request merged**